flexget/entry.py

```diff
@@ -10,6 +11,23 @@
 logger = logger.bind(name='entry')
 
 
+class EntryState(Enum):
+    ACCEPTED = 'accepted'
+    REJECTED = 'rejected'
+    FAILED = 'failed'
+    UNDECIDED = 'undecided'
+
+    @property
+    def color(self) -> str:
+        return {self.ACCEPTED: 'g', self.REJECTED: 'r', self.FAILED: 'R', self.UNDECIDED: 'd',}[
+            self
+        ]
+
+    @property
+    def log_color(self) -> str:
+        return f'<{self.color}>{self.value.upper()}</>'
```

Comment:
Not sure if it's worth putting this in here. The name is sorta confusing if we do leave it. Maybe `log_markup`?

Reply:
Agreed on the name. I kind of like adding these enhancements to enums, I use it a lot in other projects. Seems more logical in this case than to hold a dict of value to colors somewhere else